packages/wrangler/package.json

```
@@ -122,6 +122,7 @@
     "check:type": "tsc",
     "bundle": "node -r esbuild-register scripts/bundle.ts",
     "build": "npm run clean && npm run bundle",
+    "build-watch": "npm run clean && npm run bundle -- --watch",
```

```suggestion
    "watch": "npm run clean && npm run bundle -- --watch",
```

wanna make this a short word? it'll only be used by us, so let's make it easy

---

This was just to keep parity with `test-watch` below. Imo `:watch` should be a suffix you can apply to a bunch of commands. If we wanted to keep this short, maybe calling it `dev` would be better?